controllers/capa_config_map_test.go

```diff
@@ -0,0 +1,362 @@
+package controllers_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	capa "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+	capi "sigs.k8s.io/cluster-api/api/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/aws-crossplane-cluster-config-operator/controllers"
+)
+
+var _ = Describe("ConfigMapReconcilerCAPA", func() {
+	var (
+		ctx context.Context
+
+		accountID   string
+		identity    *capa.AWSClusterRoleIdentity
+		capaCluster *capa.AWSCluster
+		capiCluster *capi.Cluster
```

Comment:
Let's please keep the naming consistent with operators (including upstream projects):
```suggestion
		awsCluster *capa.AWSCluster
		cluster *capi.Cluster
```

Reply:
Done